index.html

```diff
@@ -9,12 +9,39 @@
 <body>
     <header>
         <nav>
-            <div class="hamburger-wrapper"><img class="hamburger" src="images/Hamburger.png" alt="Hamburger butten image"></div>
+            <div class="hamburger-wrapper">
+                <img class="hamburger" src="images/Hamburger.png" alt="Hamburger butten image">
+            </div>
             <div class="vertical-divider"></div>
-            <div class="count-label-wrapper"><p class="count-label">2</p></div>
-            <div class="cart-wrapper"><img class="cart" src="images/cart.png" alt="Cart image"></div>
-            <div class="logo-wrapper"><p class="logo">Renome</p></div>
+            <div class="count-label-wrapper">
+                <p class="count-label">2</p>
+            </div>
+            <div class="cart-wrapper">
+                <img class="cart" src="images/cart.png" alt="Cart image">
+            </div>
+            <div class="logo-wrapper">
+                <p class="logo">Renome</p>
+            </div>
         </nav>
     </header>
+    <main>
+        <section>
+            <div class="carousel">
+                <div class="carousel__background-image-wrapper">
```

Comment:
A word background in your class name makes me confused. Why is it needed? It will be made more sense for me if you call it `class="carousel__image-wrapper`